### PR TITLE
Update ngtf bridge doc for generic versioning and links to PyPI

### PR DESCRIPTION
**Pull request by indie**

### CODE REVIEW

**Comment by indie** [comment]
These instructions save users who are seeking the easiest option to get started a click from going to the TF repo.

File: doc/sphinx/source/frameworks/tensorflow_connect.rst [hunk 1]
```diff
@@ -1,14 +1,60 @@
...
 
+The easiest way to get started is to use the latest PyPI `ngraph-tensorflow-bridge`_,
+which has instructions for Linux* systems, and tips for users of Mac OS X. 
 
+You can install TensorFlow and nGraph to a virtual environment; otherwise, the code 
+will install to a system location.
+
+.. code-block:: console
+   
+   pip install tensorflow
```
Comment (shresthamalik) [inline doc/sphinx/source/frameworks/tensorflow_connect.rst]: Please mention the version of tf to be installed via pip (like in the bridge's readme). We might run into issues due to version mismatch between TF and ng-tf-bridge
https://github.com/tensorflow/ngraph-bridge

Note for ng-tf-bridge team, update this doc when upgrading the tf version

File: doc/sphinx/source/frameworks/tensorflow_connect.rst [hunk 2]
```diff
@@ -1,14 +1,60 @@
...
+
+Output will look something like:
+
+:: 
+
+    TensorFlow version:  [version]
+    nGraph bridge version: b'[version]'
+    nGraph version used for this build: b'[version-rc-hash]'
+    TensorFlow version used for this build: v[version-hash]
+    CXX11_ABI flag used for this build: 1
```
Comment (shresthamalik) [inline doc/sphinx/source/frameworks/tensorflow_connect.rst]: The CXX11_ABI flag could be 0 or 1 .

**Comment by indie** [comment]
Refactoring the Table of Contents (ToC) so the file affected by this PR creates less thrash.

**Pull request closed without merging**